lib/page/needed.rb

```diff
@@ -0,0 +1,31 @@
+module Page
+  class Needed
+    def initialize
+      make_client
+    end
+
+    def to_find
+      @client.issues 'everypolitician/everypolitician-data', labels: 'New Country,To Find'
+    end
+
+    def to_scrape
+      @client.issues 'everypolitician/everypolitician-data', labels: 'New Country,To Scrape'
+    end
+
+    def to_finish
+      @client.issues 'everypolitician/everypolitician-data', labels: 'New Country,3 - WIP'
+    end
+
+    private
+
+    def make_client
+      if (token = ENV['GITHUB_ACCESS_TOKEN']).to_s.empty?
```

Comment:
The Page classes shouldn't be aware of the outside world, which includes ENV. You should pass this to the constructor instead.